src/util/prop-definition.js

```diff
@@ -72,26 +94,28 @@ export default class PropDefinition {
           this[option] = optVal;
           break;
         default:
-          error(`${option} is not a valid option. Options are: attribute, initial, default, coerce, deserialize, serialize.`);
+          deprecated(`${option} is not a valid option. Options are: attribute, initial, default, coerce, deserialize, serialize.`);
```

Comment:
I wouldn't deprecate at all without a path forward. I see this as pretty low priority in the long run. Maybe we can just leave the old behaviour?